src/reducers/sources.js

```diff
@@ -56,7 +56,8 @@ export const SourceRecordClass = new I.Record({
   text: undefined,
   contentType: "",
   error: undefined,
-  loadedState: "unloaded"
+  loadedState: "unloaded",
+  tabExists: false
```

Comment:
I think we'll want to do this with either a redux selector or a check in the component

Reply:
Thanks for the early feedback, part of my motivation for throwing this out as WIP early : ).

That was my initial thought as well, but was running into some headaches with the augmenting the Source record. 

I still have that code, so I can pivot back np.